presto-hive/src/main/java/io/prestosql/plugin/hive/HiveClientConfig.java

```diff
@@ -801,6 +802,19 @@ public HiveClientConfig setOrcDefaultBloomFilterFpp(double orcDefaultBloomFilter
         return this;
     }
 
+    public boolean isOrcWriteLegacyVersion()
+    {
+        return orcWriteLegacyVersion;
+    }
+
+    @Config("hive.orc.writer.use-legacy-version-number")
+    @ConfigDescription("Use a writer version number that is readable by early Hive 2.x releases")
```

Comment:
Should this be "2.3 and earlier releases"?

Reply:
@dain i agree; especially that I _think_ problem existed in Hive 1.2 as well 

@electrum "*early* Hive 2.x releases" seems to imply "you are unlikely to need this", whereas currently _every_ Cloudera customer is affected, as their _latest_ distro still ships Hive 2.1.1 (AFAIK CDH 6.1.1 is the latest)

Reply:
I confirmed that the files are readable by Hive 1.2.1, so it's only Hive 2.x, but I agree that this should be more explicit. I'll change it to

> Write ORC files with a version number that is readable by Hive 2.0.0 to 2.2.0